packages/wrangler/src/dev.tsx

```diff
@@ -707,6 +684,12 @@ async function validateDevServerSettings(
 	const getLocalPort = memoizeGetPort(DEFAULT_LOCAL_PORT);
 	const getInspectorPort = memoizeGetPort(DEFAULT_INSPECTOR_PORT);
 
+	// Our inspector proxy server will be binding to the result of
+	// `getInspectorPort`. If we attempted to bind workerd to the same inspector
+	// port, we'd get a port already in use error. Therefore, generate a new port
+	// for our runtime to bind its inspector service to.
+	const getRuntimeInspectorPort = memoizeGetPort();
```

Comment:
Are we certain these Node processes are getting cleaned up after use?

Reply:
My concern is creating tons of zombie Node processes holding onto more and more ports.